### ISSUE TITLE
can't run on ios

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#2934]
I followed this tutorial.
http://godotengine.org/projects/godot-engine/wiki/Exporting_for_iOS

I donwloaded ``10/30/2015 12:25 AM`` templates, unzipped ``GodotiOSXCode.zip`` and opened it with xcode.
And just run, then xcode shows error.

```
Ld /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos/godot_iosTests.xctest/godot_iosTests normal armv7
    cd /Users/volzh/godot_ios_xcode
    export IPHONEOS_DEPLOYMENT_TARGET=7.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch armv7 -bundle -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -L/Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos -F/Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos -F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk/Developer/Library/Frameworks -F/Applications/Xcode.app/Contents/Developer/Library/Frameworks -F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks -F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk/Developer/Library/Frameworks -filelist /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Intermediates/godot_ios.build/Debug-iphoneos/godot_iosTests.build/Objects-normal/armv7/godot_iosTests.LinkFileList -dead_strip -bundle_loader /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos/godot_ios.app/godot_ios -framework XCTest -fobjc-arc -fobjc-link-runtime -miphoneos-version-min=7.0 -framework XCTest -framework UIKit -framework Foundation -Xlinker -dependency_info -Xlinker /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Intermediates/godot_ios.build/Debug-iphoneos/godot_iosTests.build/Objects-normal/armv7/godot_iosTests_dependency_info.dat -o /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos/godot_iosTests.xctest/godot_iosTests

ld: file not found: /Users/volzh/Library/Developer/Xcode/DerivedData/godot_ios-gpkjhoepmeuqtfhfclcmnpteynbn/Build/Products/Debug-iphoneos/godot_ios.app/godot_ios
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by volzhs** [comment]
I guess this is duplicated issue with #1790.
so I closed it.

**Comment by punto-** [comment]
We need a patch for that xcode project to delete the "godot_iosTests"
target, and also add placeholder icons and splash screens for all the
screen sizes that need to be there (especially the ones that won't
initialize the screen in the correct size if they're not present, like the
retina screens).

**Comment by volzhs** [comment]
@punto- will it be fixed at 2.0 stable? I hope so. :)
